examples/exporting_for_inference.ipynb

```diff
@@ -443,7 +443,7 @@
     "\n",
     "Some of our algorithms alter the model architecture. For example, [SqueezeExcite][squeezeexcite] adds a channel-wise attention operator in CNNs and modifies the model architecure. Therefore, we need to provide a function that takes the mode and applies the algorithm before we can load the model weights from a checkpoint. The functional form of SqueezeExcite does exactly that, and we pass this function in the `surgery_algs` argument to the `export_for_inference` API. \n",
     "\n",
-    "[docs]: https://docs.mosaicml.com/en/stable/api_reference/composer.utils.inference.html\n",
+    "[docs]: https://docs.mosaicml.com/en/latest/api_reference/generated/composer.utils.export_for_inference.html\n",
```

Comment:
What do latest and stable each respectively point to? I'm wondering if this should remain pointed at stable.